IA  mapping

Rajakavitha1 opened the pull request:
<!--
Thanks for sending a pull request! 
If this is your first time, please read our contributor guidelines:
https://docs.seldon.io/projects/seldon-core/en/latest/developer/contributing.html
-->

**What this PR does / why we need it**:

I have done a 1:1 mapping of the existing content with the [proposed new IA](https://www.notion.so/CORE-2-DOCs-IA-15a6a4c885208017bf40fb7115b4d972)

**Special notes for your reviewer**:

Commit:
summary.md ia mapping

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/SeldonIO/seldon-core?pullRequest=6236) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/SeldonIO/seldon-core?pullRequest=6236) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: paulb-seldon<br/>:white_check_mark: Rajakavitha1<br/>:x: Rakavitha Kodhandapani<br/><hr/>**Rakavitha Kodhandapani** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/SeldonIO/seldon-core?pullRequest=6236) it.</sub>

Rajakavitha1 committed:
Update SUMMARY.md

removed the orphaned topics
- [Outlier Detection]
- [Drift Detection]
- [Explainers]

Because they have a place in the new IA under:
- Data Science Monitoring
    - Dataflow with Kafka
    - Request & Response Logging
    - Model Performance Metrics
    - Drift Detection
    - Outlier Detection
    - Explainability

paulb-seldon commented:
Hey Rajie I noticed the 'Scaling' section is currently under pipelines, can you put that section up a level, outside of the pipeline section?

Rajakavitha1 committed:
Update SUMMARY.md

- moved scaling one level up

Rajakavitha1 committed:
Update SUMMARY.md

incorporated the changes that Paul suggested.

Rajakavitha1 committed:
Update SUMMARY.md

Found a place for experiments

Rajakavitha1 committed:
Update SUMMARY.md

Commit:
incoporate pauls suggestions

Commit:
moved one level up for some topics under integrations

Commit:
is changes paul suggestions

Rajakavitha1 committed:
Merge branch 'v2' into ia-rajie

Commit:
paul's changes

Commit:
incorporate Paul's feedback

Commit:
intro page changes

Commit:
fix the API section

Commit:
commented out the emplty pages

Commit:
fixed a typo

paulb-seldon committed:
Move Installation section up a level

paulb-seldon committed:
Hide concepts
